features/preferences/impl/src/main/kotlin/io/element/android/features/preferences/impl/developer/DeveloperSettingsPresenter.kt

```diff
@@ -77,6 +77,7 @@ class DeveloperSettingsPresenter @Inject constructor(
             appPreferencesStore.getTracingLogLevelFlow().map { AsyncData.Success(it.toLogLevelItem()) }
         }
         val tracingLogLevel by tracingLogLevelFlow.collectAsState(initial = AsyncData.Uninitialized)
+        val tracingLogPacks by appPreferencesStore.getTracingLogPacksFlow().collectAsState(emptySet())
```

Comment:
There is only 3 packs for now, but maybe better to use a sorted alphabetically list instead of a set?